### PR TITLE
Update Keychron Q1v1

### PR DESCRIPTION
**Pull request by adophoxia**
## Description
Superseeds #21406, but this will only cover one board from each series update of PRs. After this is made, 21406 will be closed to make things more simpler.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/keychron/q1v1/q1.c
Comment (drashna) [inline keyboards/keychron/q1v1/q1.c:1]: This needs to be renamed to match the folder name, or it won't be picked up.

**Review by drashna — CHANGES_REQUESTED** [review]
needs an entry in `/data/mapping/keyboard_aliases.hjson`

File: keyboards/keychron/q1v1/config.h [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+/* DIP switch */
+#define DIP_SWITCH_MATRIX_GRID  { {0,1} }
+
+/* Disable DIP switch in matrix data */
+#define MATRIX_MASKED
+
+/* Encoder Configuration*/
+#ifdef ENCODER_ENABLE
+    #define ENCODER_DEFAULT_POS 0x3
+#endif
```
Comment (drashna) [inline keyboards/keychron/q1v1/config.h:33]:
```suggestion
#define ENCODER_DEFAULT_POS 0x3
```

**Comment by adophoxia** [comment]
Since the LED driver for this board uses `is32fl3733`, do I also need to have: 
```h
#define DRIVER_ADDR_1 IS31FL3733_I2C_ADDRESS_GND_GND
#define DRIVER_ADDR_2 IS31FL3733_I2C_ADDRESS_GND_VCC
```
as well?

**Review by drashna — COMMENTED** [review]
If you could, yeah?

**Comment by adophoxia** [comment]
Added `IS32FL3733_I2C_ADDRESS` names in recent commit.

**Review by drashna — APPROVED** [review]

**Pull request merged**